addons/sale/migrations/10.0.1.0/pre-migration.py

```diff
@@ -92,20 +92,20 @@ def warning_update_module_names_partial(cr):
         SELECT id
         FROM ir_model_fields
         WHERE model IN ('res.partner', 'product.template') AND name in %s
-    """, (moved_fields,))
+    """, (tuple(moved_fields),))
     field_ids = [r[0] for r in cr.fetchall()]
```

Comment:
Better to make here `fields_ids = tuple([r[0]...` and you avoid to put it in all the places where it's used.